daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/ContractKeySpec.scala

```diff
@@ -378,21 +381,35 @@ class ContractKeySpec
         rollbackGlobalArchiveUpdates,
         rollbackGlobalArchivedLookup,
         rollbackGlobalArchivedCreate,
+        rollbackExerciseCreateLookup,
+      )
+
+      val nonUckFailures = Set(
+        "RollbackExerciseCreateLookup"
       )
 
       val uckFailures = Set(
         "CreateOverwritesLocal",
         "CreateOverwritesKnownGlobal",
         "LocalArchiveOverwritesKnownGlobal",
+        "GlobalArchiveOverwritesKnownGlobal1",
+        "RollbackGlobalArchiveNonRollbackCreate",
+        "RollbackCreateNonRollbackGlobalArchive",
         "RollbackCreateNonRollbackFetchByKey",
         "RollbackFetchByKeyRollbackCreateNonRollbackFetchByKey",
         "RollbackFetchByKeyNonRollbackCreate",
+        "RollbackGlobalArchiveNonRollbackCreate",
+        "RollbackGlobalArchiveUpdates",
```

Comment:
@andreaslochbihler-da could you double check those new errors are expected ?

Reply:
Indeed, these scenarios are expected to fail in Strict mode. Can you update the comments on the choices accordingly?